app/models/appointment.rb

```diff
@@ -27,7 +27,7 @@ class Appointment < ApplicationRecord
   validates :status, presence: true
   validates :guider, presence: true
 
-  validate :not_within_two_business_days
+  validate :not_within_two_business_days, unless: :agent_is_resource_manager?
```

Comment:
Haha, I think maybe it's a bit complex though. Gonna try think how to make it sound better.